BT-Tracking/Interface/Active/ContactsController.swift

```diff
@@ -25,7 +25,10 @@ final class ContactsController: UIViewController {
     }
 
     @IBAction private func faqAction() {
-        openURL(URL: URL(string: "https://koronavirus.mzcr.cz/otazky-a-odpovedi/")!)
+        let remoteConfigValue = AppDelegate.shared.remoteConfigString(forKey: RemoteConfigValueKey.faqLink)
```

Comment:
Idealne, bych k tem hodnotam chtel pristupovat podobne jak to je v AppSettings, pres nakou struct a variables, at nemusi byt pri kazdem "get" tak dlouhy kod.

Reply:
@augard  Můžeme je rovnou dát do AppSettings (A):
```
    static var faqLink: String {
        return AppDelegate.shared.remoteConfigString(forKey: RemoteConfigValueKey.faqLink)
    }

```

a pak přistupovat např. takto:  `URL(string: AppSettings.faqLink)`.

A nebo vytvořit novou strukturu (B) např. `RemoteValues` (`RemoteValues.faqLink`).
A nebo B?

Reply:
Jsem radeji pro B, diky